src/screens/ProfileScreen/ProfileScreen.js

```diff
@@ -1,21 +1,238 @@
-import React, { Component } from 'react';
-import { View, Text, TouchableOpacity } from 'react-native';
-import styles from './styles';
+import React, { useState, useEffect } from 'react';
+import { View, Text, TextInput, TouchableOpacity, SafeAreaView, Image, ScrollView, Platform, Alert } from 'react-native';
 import { firebase } from '../../config/config';
+import ImagePicker from 'react-native-image-crop-picker';
+import styles from './styles';
+import { LOGO, NO_IMAGE } from '../../config/styles.js';
```

Comment:
Here you are importing NO_IMAGE which I can not find it in styles file. Might missed when committing.